packages/contentstack-migration/examples/change-master-locale/02-change-master-locale-new-file-structure.js

```diff
@@ -0,0 +1,158 @@
+        }
+        locales[id] = {};
+        locales[id].uid = id;
+        locales[id].code = masterLocale;
+        locales[id].name = supportedLocales[masterLocale];
+        locales[id].fallback_locale = config.target_locale;
+
+        await handleEntries(masterLocale);
+        await fs.writeFile(
+          pathValidator(path.resolve(config.data_dir, 'locales/locales.json')),
+          JSON.stringify(locales),
+        );
+        masterLocale = await fs.readFile(
+          pathValidator(path.resolve(config.data_dir, 'locales/master-locale.json')),
+          'utf-8',
+        );
+        masterLocale = JSON.parse(masterLocale);
+        const uid = Object.keys(masterLocale);
+        masterLocale[uid].code = config.target_locale;
+        masterLocale[uid].name = supportedLocales[config.target_locale];
+        await fs.writeFile(
+          pathValidator(path.resolve(config.data_dir, 'locales/master-locale.json')),
+          JSON.stringify(masterLocale),
+        );
+      }
+
+      async function handleEntries(masterLocale) {
+        let contentTypes = await fs.readdir(pathValidator(path.resolve(config.data_dir, 'entries')));
+        for (let contentType of contentTypes) {
+          let sourceMasterLocaleEntries, targetMasterLocaleEntries;
+
+          sourceMasterLocaleEntries = await fs.readFile(
+            pathValidator(path.resolve(config.data_dir, `entries/${contentType}/${masterLocale}/index.json`)),
+            { encoding: 'utf8' },
+          );
+
+          sourceMasterLocaleEntries = await fs.readFile(
+            pathValidator(
+              path.resolve(
+                config.data_dir,
+                `entries/${contentType}/${masterLocale}/${Object.values(JSON.parse(sourceMasterLocaleEntries))}`,
+              ),
+            ),
+            { encoding: 'utf8' },
+          );
+          sourceMasterLocaleEntries = JSON.parse(sourceMasterLocaleEntries);
+          if (
+            existsSync(pathValidator(path.resolve(config.data_dir, `entries/${contentType}/${config.target_locale}`)))
+          ) {
+            targetMasterLocaleEntries = await fs.readFile(
+              pathValidator(path.resolve(config.data_dir, `entries/${contentType}/${config.target_locale}/index.json`)),
+              { encoding: 'utf8', flag: 'a+' },
+            );
+          }
+
+          if (targetMasterLocaleEntries) {
+            targetMasterLocaleEntries = await fs.readFile(
+              pathValidator(
+                path.resolve(
+                  config.data_dir,
+                  `entries/${contentType}/${config.target_locale}/${
+                    Object.values(JSON.parse(targetMasterLocaleEntries))[0]
+                  }`,
+                ),
+              ),
+              { encoding: 'utf8' },
+            );
+            targetMasterLocaleEntries = JSON.parse(targetMasterLocaleEntries);
+          } else {
+            targetMasterLocaleEntries = {};
+          }
```

Comment:
@cs-raj [Minor] we can move if & else condition after L95

Reply:
Moved